### DISCUSSION TITLE
posts/2024/05/body-for-soul/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#60] [General]
# posts/2024/05/body-for-soul/

https://guozheng.rbind.io/posts/2024/05/body-for-soul/

**Comment by earfanfan** [comment]
从俺的视角看，这首诗写得并不矫情。当然，如果非要批判一下的话，也是可以批判的。

不过，下次觉得“矫情”的时候，不妨换个角度看看，比如感到“矫情”也许是一种情绪，试着面对这种情绪，寻找情绪产生的原因（PS前面的讨论中看到你好像已经找到了），而不是顺着情绪在心湖的肆意游走，搅起风浪进而批判自己。

对了，你提到的那几篇文章我还没来得及看，倒是通过那个公众号又看到另一个公众号去了，叫新潮沉思录，感觉好像是无意中又打开了一扇新世界的大门，这让我觉得好神奇啊。前些日与你的交流中，让我有一瞬间发现，原来我已经变成了三年前自己想要成为的样子，当年我对自己有许多批判，但列弛给予了我许多鼓励，如今我居然在鼓励你，哈哈哈。

**Reply by residualsun1** [reply]
> 前面的讨论中看到你好像已经找到了

嗯，我觉得前面提到的“不要表演悲伤”就是一个很好的方向。其实益辉之前也写过类似的内容，我记得好像是说不要沉浸在羞愧里——比失败更可怕的是沉浸在羞愧中。

---

我在前年的时候关注了“新潮沉思录”，当时网络上有很多关于疫情舆论的讨论，公众号写了一篇《[今日欢呼孙大圣，只缘妖雾又重来](https://mp.weixin.qq.com/s/3K4sgOB3Bn_RS1IHl-Qrew)》，斥责了一些媒体人。最近在这篇关于阿里数学竞赛的文章里，作者也批评了媒体。

说起来，我本来还在关注阿里数学竞赛的事情，但在豆瓣上看到这事情变得越来越复杂，什么声音都有，新冒出来的声音都可疑地想把话题引到其他方向，然后又有许多网民跟过去，我就不想再了解下去了。网络上的声音七零八乱，让人心神不宁。

---
> 让我有一瞬间发现，原来我已经变成了三年前自己想要成为的样子

;D

**Reply by earfanfan** [reply]
我就是看到你给的那个公众号转载了新潮沉思录写姜萍的文章，才决定关注的。文章里提到的一点让我深受震撼，大意是初中中考是一次教育分流，但是国家并不重视职业教育的发展，导致职校的学生备受歧视，而我在看到这篇文章之前，脑子里并未思考过这点，所以原本仍然是充斥着类似的刻板印象。

又由于脑子里的刻板印象更新过了，后面我在整理教育部的教育统计数据时，顺带看了里面一些职业教育方面的学生数据，才发现原来2008年的时候，里面竟然统计了一些[职业学校学生变动的原因](http://www.moe.gov.cn/jyb_sjzl/moe_560/s4628/s4629/201010/t20101021_109952.html)，恰好几个月前我统计过我国的一些自杀率数据，感觉那年职校学生的死亡率是高过同年龄段几倍的，从我的视角看，要特意统计这些数据，一定是这个问题已经很严重需要官方介入了。

至于姜萍这个女孩，自从看到她的新闻，我从未质疑过她，也没有看过任何质疑她的内容，后续会继续关注。

**Reply by CyrusYip** [reply]
> 至于姜萍这个女孩，自从看到她的新闻，我从未质疑过她，也没有看过任何质疑她的内容，后续会继续关注。


@earfanfan  姜萍的新闻好热门，连我这位数学门外汉都知道了。目前的进展是多名参赛选手要求调查姜萍。

- [如何看待阿里数赛多名选手联名请愿，要求组委会就「姜萍事件」展开第三方独立调查？ - 知乎](https://www.zhihu.com/question/659628342)

**Reply by residualsun1** [reply]
> 目前的进展是多名参赛选手要求调查姜萍。

好像有点两极分化，豆瓣上多数人反对质疑，但在知乎和bilibili上却有不少质疑声。让子弹再飞一会儿吧。

**Reply by earfanfan** [reply]
@CyrusYip  叶寻同学，你的博客更新得有点少呀，俺趁此机会催耕一下。

@residualsun1  黄同学，是不是放暑假了，博客耕起来呀。

其实我说的关注姜萍同学并不是在关注最近的事件有什么进展，而是关注她的人生将来会不会有什么好的发展。从我的视角看，如此大声量的质疑和审判恐怕已经改变了她的人生，希望她能挺过来，然后获得和普通人一样的幸福和满足。

对了，我有催耕两位，两位当然也可以催耕我，俺最近也继续坚持在写，只不过卡关，所以慢……

**Reply by residualsun1** [reply]
> 黄同学，是不是放暑假了，博客耕起来呀。

@earfanfan 今天下午终于考完最后一门期末科目了，虽然学校统一7月13放，但是各学院考完试=放假哈哈 :-D。没耕博客是因为近期忙于世俗（bushi，事实上还在修改）

另，我看了一下["中等职业学校（机构）学生变动情况"](http://www.moe.gov.cn/jyb_sjzl/moe_560/s4628/s4629/201010/t20101021_109952.html)的数据，不知道你之后会不会做相关的数据分析，蹲一蹲。我很久没学R了，但看你博客的时候都会有点心痒痒。

> 对了，我有催耕两位，两位当然也可以催耕我，俺最近也继续坚持在写，只不过卡关，所以慢……

之前看你博客，我推测是有某个月会集中某个时间段发得比较多，然后下个月就会显著减少，所以默认有你自己的写作规律，下回我也催催……

@CyrusYip 滴滴，更一下我今天看到的文章《[我国著名数学家北京大学袁新意教授评论姜萍事件](https://mp.weixin.qq.com/s/hFhbEPGje0ymIr0aF4iP6g)》，还是27日下午发布的。我觉得袁教授的分析是很冷静的，不论最后结果如何，都本着保护姜萍的出发点，分享一下。

**Reply by CyrusYip** [reply]
@earfanfan 最近在对博客做大量琐碎的修改，得改完才开始发文章。

@residualsun1 袁新意教授的文章很好，既专业又有温度。

**Reply by earfanfan** [reply]
额，讲真，我觉得叶寻同学和黄同学都有点天真，这里不是鄙视你俩认知浅薄，就是真得觉得你们有点天真。我看了袁新意老师写的文章后，感觉他也有可能成为下一个被网暴的人，我的想法如下。

从我的视角看袁老师不太了解网络世界现在蛰伏着多么可怕的怪物，他应该只是看过新闻知道网暴多可怕，但是从没亲身经历过。我也觉得他确实是很冷静地提出了自己的质疑，但是质疑他的质疑的人，也还是可以从一些角度攻击他。

比如，他写过“一个中专生……”，这样的话从我的视角看，想要表达的是对比中专生、本科生、研究生的不同平均水平，这是一种客观的社会现象，但依然可以被解读成一种刻板印象，甚至延伸为主观的歧视。

又如，袁老师以及许多看过他文章的人都认为他质疑的出发点是冷静的，但是如果换成是预设了他的出发点是带着情绪或者带着目的的人，这些人就可以理所应当认为“以情绪对情绪”是没问题的，这种情况下，如果袁老师继续试图向这些人自证清白，也是没用的。毕竟他也没法控制别人能从他的文字中解读些什么出来。

再者，就是袁老师提及的团队作弊方式，质疑他的质疑的人，完全可以进一步质疑，既然这种团队作弊方式存在，那么姜萍被质疑是否是因为她的学历或性别或年龄或其他相对弱势的方面，而那些清北学子是否因为学历等方面而从未有人质疑作弊。当然我也看过一些文章，大意是指责阿里数赛的初赛存在范围很广的团队作弊。一旦“质疑”的动机被指责，这个也是很难辩得清的。而且袁老师那篇文章结语的落点是“还大家一个公道，还数学界一个清白，还数学界一片净土”，这样的文字也很容易延伸出许多令人意外的内容啊，比如不同的人想要的公道不一样的话就……

从我的视角看，其实很多人是借着关心热点事件的真假来表达自己的诉求的，而人们之所以会这么做，更可能是因为生活中充满了无可奈何，比如贫寒学子面对城乡教育资源的不平均分配感到无可奈何，又比如贫富差距日益扩大的当下人们对苦苦求生尊严仍被践踏感到无可奈何，又又又如性别对立愈发严重的网络世界中许许多多的无可奈何。

**Reply by residualsun1** [reply]
> 从我的视角看，其实很多人是借着关心热点事件的真假来表达自己的诉求的，而人们之所以会这么做，更可能是因为生活中充满了无可奈何，比如贫寒学子面对城乡教育资源的不平均分配感到无可奈何，又比如贫富差距日益扩大的当下人们对苦苦求生尊严仍被践踏感到无可奈何，又又又如性别对立愈发严重的网络世界中许许多多的无可奈何。

@earfanfan 嗯，我能接收到并赞成你的想法。在看到你的留言前，我从没想过袁老师是否会有被网暴的风险，目前还没发现。微信上还好，豆瓣上则主要是很多骂质疑姜同学的人的声音，我回豆瓣又看了一会儿，觉得对“表达自己的诉求”这一点会有一些体会，同时也觉得像你说的那样，互联网上有蛰伏的“怪物“（其他平台应该也有，但我用豆瓣较多）。我没经历过网暴，还没真切体验过它给人带来的伤害，但互联网将人性放大的恶也能让我隔着屏幕都觉得可怕。

但心里却还是会因为有相对理性声音的出现而受到鼓舞。不过和你说的公道那般，我支持袁教授建议姜同学表现水平，并以自己的方式给予姜同学帮助是一种理性的做法，但也有很多网友认为这是在逼小六子证明凉粉，或者说看不得中专女孩的优秀等等——每个人的标准都不同，那么不同的标准又要怎么协调？我也不知道标准之后有没有标准，这标准又是什么模样……按数学界、科学界看法，面对质疑是自然的事情，且打破质疑更是对自身实力的有力证明；按民间部分看法，要姜同学证明自己，实质是有权之人对无权之人的欺压。这应该也是一个很好的社会学思考议题，但我还无力解释……

以上我认为都是可以理解的，并非理解事情的内容和实在，而是它们存在的理由。但我难以理解的是为什么人和人之间的恶意那么容易触发呢，而且这恶意肆意增长……或许你最后写的那一段是一种答案，在我归纳看来便是人们将在现实中遭遇到的不公、压迫转而在网络上以另外的形式表达出来。不过，唉，这也是我的幻想，我还是会天真地以为人们本可以不如此，在我眼中平凡的老百姓不会那么热衷于纠结于此，党同伐异、相互攻击。

**Reply by earfanfan** [reply]
@residualsun1  

也许是黄同学你还没经历过利益相争的事，没有近距离体会过“天下熙熙，皆为利来，天下攘攘，皆为利往”，所以看到的世界大多是阳光美好的那面，等到将来有机会凝视过深渊以后，可能才会明白一些别的，比如“魔鬼之路，有时是由善意铺成的”，世间的“正义”未必只有一种，有些自以为是的正义也是正义。

在我继续展开之前，我先要跟你说另一件事，虽然我比你大很多，但你可千万不要对我有什么滤镜，就是那种误以为人类的认知都会随年龄拓宽或者迭代之类的，其实我最近一直都很迷茫，困惑于应该选择什么样的态度来看待世间一切。比如我看另一位黄同学写的文章[《我负丹青 · 吴冠中自传》](https://xiangyun.rbind.io/2024/05/wu-guan-zhong/)，其实我看完后脑中想了一些关于特殊历史时期的其他角度，比如“合法性基础”、“党管思想”、“政治虽然各层级同构但并非上下一体”等许多，我说要展开写写，可是一直卡关，就是因为难以捋出来一条主线能够写尽所有所思所想。

关于围绕姜萍同学的事，我也是困惑的，并不能够做到坚定地做出唯一的选择，只是因为自己曾经被别人审判过，经年日久才看开，希望她能少受些伤害。我仔细回忆了一下袁老师那篇文章，只凭自己的感觉的话，感觉他是把姜萍同学当做一个平等的个体来对待的，应该是真得没有意识到他的相对较高的社会地位会给自己的文字带来多少加成（PS还有一种可能他的本意只是想写出来，也没想到会被大量媒体和自媒体转载造势），科学界确实是推崇面对质疑的，但是姜萍同学被质疑的事其实该算是她的生活中的事。我以前看过一张图，大意是在一个长方形中，左边三分之一被填满红色，右边三分之二被填满蓝色，如果在中间二分之一位置划一条线的话，这条线到底算是中立，还是在势力完全不对等的情况下的中立并非真正的中立。

如果你把这些事当做一个社会学思考议题的话，我猜你的老师会给你讲，喂，先观察，不要先想着去解释，免得预设了因果逻辑，反而困住了自己的思想。

另外，思考的时候，用二元框架来想只是一种思路，其实应该还有别的角度的，比如在你看来是恶意的内容在发言之人自己看来是正义，又比如也许这个时代正在由人民自己重新定义善恶的边界（PS这点要展开的话就不得不涉及政治因素，暂略），既然是重新定义的话，新的边界往哪个方向发展尚未可知，如果是更多元的变化方向的话，避免不了会撕裂，而撕裂未必就百分百是坏事，因为时间是流动的，未来依然有融合的可能，或者互不认同但相互理解与尊重的可能，朝这个方向继续想的话，又不得不说说话语权大小的问题，还有流量时代的其他特色，但是我明早要赶火车，也不展开了。另外，还有一个思路，恶意的触发也有可能是别有用心的引导，为了蹭到流量，你应该也看过姜萍家的鸡成网红之类的内容，继续展开的话，就又得说起，唉，大家怎么那么容易被带情绪咧，或者难道说多数人是缺乏独立思考能力，别人灌什么思想就变成什么被扯线的木偶么……啊，快十一点了，我困得有点头晕了也

**Reply by CyrusYip** [reply]
@earfanfan 你想得好宽泛。我认为这就是个学术伦理问题。我只关注事件真相——姜萍的数学水平与是否作弊，吵架、蹭热度的报道、借题发挥这些我就忽略了。

> 从我的视角看袁老师不太了解网络世界现在蛰伏着多么可怕的怪物，他应该只是看过新闻知道网暴多可怕，但是从没亲身经历过。我也觉得他确实是很冷静地提出了自己的质疑，但是质疑他的质疑的人，也还是可以从一些角度攻击他。
>
> （三个例子）……

你例子里面那样的人在我看来就是杠精——自己一直坚持某个观点却常常没有证据，论战失败还会骂人。我上网看过不少，但是仍然无法理解他们的心理。被杠精缠上确实很糟心，简直想骂人，如果你和他对骂就开始变得像他一样无理取闹了。

大家当然可以质疑那些质疑姜萍的人，但是不能只说观点不说证据。比如，说有人歧视中专生，就拿出证据。至少我在文章里没看到这样的歧视。没证据的质疑就是废话，不值得回应，也难以回应，证明自己没做一件事是很难的。

---

> 但心里却还是会因为有相对理性声音的出现而受到鼓舞。

@residualsun1 同感。如果讲道理的人不站出来，世界就变成谁大声谁有理了。我在网上看过太多蛮不讲理的人，搞得我很多时候我都不敢发表一些小众观点，不是担心被质疑，是担心遭遇网络暴力（我会难过）。看了袁教授的文章，我又获得了一些勇气，谢谢你分享。

---

@earfanfan 你之前在 <https://github.com/earfanfan/yuanfan/pull/609#issuecomment-2012556846> 这里说我可以质疑科技博主。其实我批评用词错误的时候想找一些具体的视频（我不是针对他们，只是觉得有具体的例子更好），但是会担心被科技博主的粉丝攻击（也有可能我想多了）。希望自己有一天能有勇气面对网络暴力。

**Reply by earfanfan** [reply]
@CyrusYip 

>希望自己有一天能有勇气面对网络暴力。

我倒是没觉得你想多了，而且是否有勇气面对网络暴力其实本来也不该只是你的问题。网络暴力的诞生有许多种，其中有一种我在 COS 论坛也见过许多次，就是大家都觉得自己是对的别人是错的，举个我觉得还算比较温和的例子，有新人问如何系统性地学习一个理论或技术或其他，新人提问的时候透露出对学习的态度是“先学习、后实践”，但是坛子里有好几个版主对学习的态度是“在实践中学习”，于是乎他们从“实践出真知”的角度各种举例子来劝楼主改变学习态度。当叶寻同学你在批评用词错误的时候，额，从我的视角看，“用词求精准”跟大众玩梗其实是反方向的，如果你是给益辉这样与你在语言表达这方面一样求精准的人提出批评，那他肯定超开心的，但是换成那种大家不追求这个的环境中就会有点容易被误解，“精益求精”会被误解成“鸡蛋里挑骨头”之类的，毕竟个别用词不影响内容表达，更像是一种态度。

不过如果你某天真得很想提出质疑，我想到的最好的办法就是，先确定环境是相对安全的，比如 COS 论坛的氛围就算比较安全的，或者其他熟一点的博主的博客园子里，或者科技博主为人温和且粉丝留言也大多温和。这几年社会上暴露出很多问题，网络上积攒了超多戾气，最近我都看到好几起无差别伤人事件了，如果不是有意追求流量当什么意见领袖的话，我会觉得先在安全点的环境中畅所欲言是舒服的。

**Reply by residualsun1** [reply]
@earfanfan @CyrusYip 我觉得我是比较害怕招致他人反对的人，这种反对的面很广，如程度比较高的质疑、明确的反对与对立和网络暴力等（某种程度上来说前两者其实有利于我重新进行思考），所以我的博文里很少会出现比较明确的判断，而“或许”、“也许”、“似乎”这样的用词在文章中也相应地会出现得很频繁。很多时候我也不敢在线上和线下的公众场合明确表达自己的看法，所以知道袁教授发声——我认为理性、良善和解决问题为中心的发言——后，我也会感到宽心。

（我可能暂时只能用到二元框架来想）

或许在叶寻眼里，袁凡想得比较宽泛，但我个人反而觉得会更有助于理解“姜萍获奖”这件事引起的争议。我认为在这里“姜萍获奖”已经不是姜萍的事情了，如同袁凡说的这是多种社会问题的集合、不同群体情感和诉求的宣泄口，不然它也不会发酵成今日这般，成为我们仨讨论的话题。

在我眼里，叶寻说的“事情的真相”是**重要**的，袁凡说的“审判”等方面的多方面思考则是**必要**的。如果只有观点、立场和情绪，双方间的讨论可能就没办法进行下去，无法尝试理解对方，所以需要一根准绳。和袁凡后面提到的“如果是更多元的变化方向的话，避免不了会撕裂，而撕裂未必就百分百是坏事，因为时间是流动的，未来依然有融合的可能，或者互不认同但相互理解与尊重的可能”不一样，目前的撕裂很多时候让我觉得“触目惊心”，有限的上网经历让我在面临撕裂初期时便感到恐惧，所以我仍然倾向一个我眼中比较理性的、权威的声音/准绳存在。（不过我觉得袁老师的出现也是让这件事情进一步“流动”了，向下一步走了）

但关于袁凡的多方面思考，我觉得应当被看见。其一，袁老师身份对文章的影响及媒体是否造势等都应该纳入考量，首先保护作为个体的姜萍。因为我们所看见的乱象、争议已经不是“姜萍”了，而是姜萍外的各种声音，但姜萍却可能要承担不应有的影响。其二，如果只考虑学术伦理问题，或许还是会困惑为什么那么多人仍然不满，放任这些不满不管，可能只会让问题继续被忽略、掩盖、发酵，于暗流中愈加剧烈地涌动，而理解大家的不满在我看来是值得尝试的，这种尝试没法立竿见影地解决每个人的问题，但可能是一种使得进一步沟通、理解出现的措施，即便这可能只是正在想这件事情的人单方面的意愿，而还在网络上争论的当事人还不知情。讲到这，我觉得可以分享袁长庚老师在书店分享会上提到他自己对性别对立的一些看法，让我觉得很触动：

> 我当老师最重要的一个事情就是学习，可以理解我学生的愤怒，尤其是比如说女孩的愤怒。有时候我也觉得不好意思，我又不是直男，我又没冒犯你，你为什么很多东西都来挑我的毛病？但后来，我觉得这个是越过不了的。**就是如果你真的去走进她们的生命时，你知道无论她今天在你面前表现得多么坚强、多么完整，她可能在某一个时间碎掉过，所以她今天愤怒是有她的权利和道理。你只能从这个愤怒出发，你只能跟着这个愤怒一起成长，你只能用你的知识先去应对这个愤怒**。

过去我一直害怕性别话题，不敢参与讨论，也从未去想象那一片世界。其实社会“沉默的大多数”可能就真的是大多数，中立、沉默、安稳看起来似乎没啥问题，但不论是性别、教育平等、贫富差距等任何社会问题，其实都需要这波人——当问题中的弱势方愤怒时，采取着有悖理性的行动时，这波人表现出的倾听和理解是否会对弱势方有所帮助呢……就我个人角度来看，当时我是受到袁长庚老师很大的触动，收获了一种我从未有过的思考该问题的方式，这种来自知识分子的坦诚很诚恳，我也相信书店分享会上倾听的观众都有所触动。因此，我觉得至少要先让众人（女性、受制度所困的穷人、教育体制中的弱势方等）的愤怒得到关于尊重的回应。回应，意味着我知道你在听我说话，而且你愿意思考我的处境，即便你没办法帮助我改变，而不是表面在听而背地里冷漠、嘲讽甚至变本加厉，我想女性尤为愤怒的一个原因很大程度在于不少男性常扮演着那样一种角色。只有感受到你对我们的尊重以后，我们才有心平气和地谈共同合作、共同改变的可能。

（其实想到集体联名质疑姜萍时，而且在阿里本身竞赛原先就有不少作弊说法，且选手都匿名，有的只放出自己学校的情况下，怎么看也觉得蛮可疑的。就学理上讲，我认可捍卫数学学术；但这些人自身的动机我会保持怀疑，我也选择去理解豆瓣上不少对此的激烈言辞。但同时，我很讨厌，很厌恶部分自媒体要给姜萍“造神”）

不止是性别。我个人认为社会上很多不公的社会现象都是这种状况，我之前在推特上看到很多农民工讨债、跳楼的视频。这些农民工某种程度上说实打实经受着赤裸裸的不公，但他们甚至连相对“温和”发声的渠道也没有，更没有什么人回应、尊重他们。

**Reply by earfanfan** [reply]
@residualsun1 

>我也选择去理解豆瓣上不少对此的激烈言辞。
>我之前在推特上看到很多农民工讨债、跳楼的视频。

小伙子很勇啊，这种跳楼视频我是不敢看的，看了晚上会做噩梦，记得以前叶寻同学也曾留言白天看恐怖片也会害怕。激烈言辞我总体上看得也极少，看到的时候也是进行“理解”，说是理解也不对，应该说是判断内容之外的立场、阶级、目的之类的。

“这些农民工某种程度上说实打实经受着赤裸裸的不公”，看到这句话，我想说说我了解到的信息。约莫是21世纪初的时候，中国刚加入世贸组织，贸易顺差带来大量红利，综合一些暂不展开的复杂原因政府开始大力发展城市基础建设，吸引许多农民离乡背井来城市打工，只不过政府花钱的模式并不是有多少钱花多少钱，而是“如果需要完成的任务是当前实力达不到而又不得不完成的任务，作为代理方的政府只能凭借被赋予的合法权力向其他组织攫取资源”，大概就是建设的时候先找施工队干活，结账的时候能拖就拖。那些年的新闻常常是包工头卷钱跑路导致农民工踏上漫漫讨薪路，但其实不少是各级政府本来也没兑现承诺。那些年还有另一些宣传，就是大力宣传农民工为城市建设奉献，把农民工捧到一个道德上至高点的位置。不过这也只是其中一个角度，但“某种程度上说”可以去掉。

>但他们甚至连相对“温和”发声的渠道也没有，更没有什么人回应、尊重他们。

其实是有发声渠道的，但是这个“有”也有门槛，很多城市都有市长热线、城市留言板之类的，又但是……其实可以把人民在互联网上宣泄情绪当做一种“无组织的集体行动”，因为一些政治原因，现实世界中人们无法做出有组织的集体行动，但是共同遭受的不公和心中的愤懑不满又使得大家的行为自发变成了一种无组织的集体行动。按照我们国家的政治制度，政府的一切权力来自人民代表大会，但是现在人民代表大会上的提案因为种种原因无法代表所有人民的诉求，而作为统一战线方式可以补充建议的政治协商制度又逐渐变得精英化（政协委员来自政界商界精英的比例偏多）。所以说发声的渠道是“有”的，只是数量众多的底层人民往往够不到那个发声的门槛，只能被代表，现实中也有些社会原因使人有苦不能诉尽，其他渠道被堵了以后，最后只好在互联网上宣泄并且变得严重情绪化。

>我觉得我是比较害怕招致他人反对的人，这种反对的面很广，如程度比较高的质疑、明确的反对与对立和网络暴力等（某种程度上来说前两者其实有利于我重新进行思考），所以我的博文里很少会出现比较明确的判断，而“或许”、“也许”、“似乎”这样的用词在文章中也相应地会出现得很频繁。很多时候我也不敢在线上和线下的公众场合明确表达自己的看法，所以知道袁教授发声——我认为理性、良善和解决问题为中心的发言——后，我也会感到宽心。

其实我也常用“或许、也许、可能、大概、似乎、好像”这类的词语，但原因不是怕被反对，而是自己也不能百分百确定某一种逻辑、某一种现象就是唯一，用这类词语可以保留一些“不确定”的质疑空间。我想了一想，自己好像并没有怕过被反对，但是也会下意识怕自己与别人不同，被当做异类，当然也是会怕那种非常暴力的强烈质疑。话说前些天我跟一个球友去另一片她熟的场地打球，那边的组织者看到我以后从上到下扫视了一眼，然后问球友（指着我）“她会打球吗？”，后来又拉着球友说我都没有穿球鞋所以我只能跟那位带我去的球友玩不能跟别人玩，那位球友跟我讲了以后，我第一反应是要去买专门打羽毛球穿的鞋免得又被歧视。不过后来我想明白了一点，不想因为不想被歧视就不穿自己觉得舒服的鞋子而买新鞋子，哈哈。

额，最后，理解别人的不满、尊重别人的愤怒都是很好的，但是黄同学你最好也学会做到课题分离，不要背负别人的人生，不然大概率会承受不住。

**Reply by residualsun1** [reply]
@earfanfan 哈哈，这不是你第一次规劝我做课题分离了，感谢建议: D.

> 不想因为不想被歧视就不穿自己觉得舒服的鞋子而买新鞋子

我表示大力支持。

另，接下来很长一段时间我都会断更博客啦，需要集中花上时间做别的事情。但无论咋说，都很开心你的催更，更增添了我码字的动力，我会保存起来，待一段时日后再拿出来。

**Reply by earfanfan** [reply]
@residualsun1

博主人还怪好的嘞，断更还会跟读者讲一声。无论你去忙撒子，希望你顺利，加油。

**Reply by CyrusYip** [reply]
https://github.com/residualsun1/Residualsun/discussions/60#discussioncomment-9889991

@earfanfan 恭喜你催更成功🎉。我发了一篇[菜谱](https://cyrusyip.org/zh-cn/posts/2024/07/05/fried-egg-recipe/)。
